moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -989,7 +1260,7 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(robot_trajectory::RobotT
   }
 
   // Now actually call the algorithm
-  Trajectory parameterized(Path(points, path_tolerance_), max_velocity, max_acceleration, 0.001);
+  Trajectory parameterized(Path(points, path_tolerance_), max_velocity, max_acceleration, DEFAULT_TIMESTEP);
```

Comment:
now I see why you added the constant. Are we actually sure that 0.001 is a good number? What if `resample_dt_` is smaller? Maybe we should set this one as a factor of `resample_dt_` to ensure adequate accuracy.